client-code/Stylesheet.html

```diff
@@ -200,4 +200,11 @@
       height: 0%;
     }
   }
+
+ /********* Projects Reload Data Button ********/
+ .reload-button-container {
+    left: 70em;
+    top: 12em;
+    position: absolute;

```

Comment:
Sorry, I should've addressed this with the previous review, but generally, I believe using absolute positioning for CSS is not the best practice. This is because the HTML object will not be part of the document flow and so if you changed something on the page, the object that used absolute positioning will still be at the same location on the page. You would have to go in and change the specific CSS for objects that use absolute positioning in order to change its location, which is not ideal as applications get more complicated. For something like this, I would use a flex container instead. There's lots of documentation online about flex containers, but I think these ones are simple and to the point:
1. https://www.w3schools.com/css/css3_flexbox_container.asp
2. https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Feel free to ask me if you have questions as well.

Reply:
Addressed by commit 66b509a3a5f1b0d665f2116d5606ba4aa0cdce64